ui/raidboss/data/06-ew/raid/p2s.ts

```diff
@@ -0,0 +1,311 @@
+      alertText: (_data, _matches, output) => output.text!(),
+      run: (data, matches) => data.flareTarget = matches.target,
+      outputStrings: {
+        text: {
+          en: 'Tank flare tether on YOU',
+        },
+      },
+    },
+    {
+      id: 'P2S Coherence Stack',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '681B', source: 'Hippokampos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '681B', source: 'Hippokampos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '681B', source: 'Hippokampos' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '681B', source: 'ヒッポカムポス' }),
+      condition: (data) => data.flareTarget !== data.me,
+      // 12 second cast, delay for tether to settle
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 6,
+      alertText: (_data, _matches, output) => output.flareLineStack!(),
+      outputStrings: {
+        flareLineStack: {
+          en: 'Flare => Line Stack',
+        },
+      },
+    },
+    {
+      // Raidwide knockback -> dont get knocked into slurry
+      id: 'P2S Shockwave',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '682F', source: 'Hippokampos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '682F', source: 'Hippokampos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '682F', source: 'Hippokampos', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '682F', source: 'ヒッポカムポス', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P2S Kampeos Harma Marker',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '00E[56789ABC]' }),
+      condition: Conditions.targetIsYou(),
+      response: (_data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          square: {
+            en: '#${num} Square',
+            de: '#${num} Viereck',
+            fr: '#${num} Carré',
+            ja: '#${num} 四角',
+            cn: '#${num} 四角',
+            ko: '#${num} 짝수',
+          },
+          triangle: {
+            en: '#${num} Triangle',
+            de: '#${num} Dreieck',
+            fr: '#${num} Triangle',
+            ja: '#${num} 三角',
+            cn: '#${num} 三角',
+            ko: '#${num} 홀수',
+          },
+        };
+
+        const id = matches.id;
+        if (!id)
+          return;
+
+        let num = parseInt(id, 16);
+        const isTriangle = num >= 233;
+        num -= 228;
+        if (isTriangle)
+          num -= 4;
+
+        // Odd numbers have to run to the other side, so make this louder.
+        // TODO: should this be alarm/alert instead of alert/info?
+        const isOdd = num % 2;
+        const text = isTriangle ? output.triangle!({ num: num }) : output.square!({ num: num });
+        return { [isOdd ? 'alertText' : 'infoText']: text };
+      },
+    },
+  ],
+  timelineReplace: [

```

Comment:
Can you add a `timelineReplace` for English to replace `Spoken Cataract/Winged Cataract` with `Spoken/Winged Cataract`?